File: Scylla/General/Models/Company/CompanyResponse.cs
using Data.Entities;
using Shared.Models.Address;
using System.Numerics;

namespace General.Models.Company
{
    public class CompanyResponse :BaseAuditCode
    {
        public Guid CompanyId { get; set; }
        public string CompanyName { get; set; }
        public string Email { get; set; }
        public string Phone { get; set; }
        public int CompanyCode { get; set; }
        public bool IsActive { get; set; }
        public ICollection<GetAddressResponse> AddressList { get; set; } = [];

    }
}

File: Web2/src/app/types/company/company-response.ts
import { AddressResponse } from "../shared/address";

export interface CompanyResponse {
    companyId: string;
    companyName: string;
    email: string;
    phone: number;
    companyCode: string;
    createdBy: string;
    CreatedDate: Date; 
    isActive: boolean;
    updatedBy: string | null;
    updatedDate: Date;
    addressList: AddressResponse[]; 
  }